src/moda_bufrsr.F

```diff
@@ -35,7 +35,7 @@ MODULE MODA_BUFRSR
 	  INTEGER :: JNOD
 C>        Section 1 date-time of BUFR message.
 	  INTEGER :: JDAT
-C>        ???
+C>        Indicator of stack status when entering subroutine rewnbf.
 	  INTEGER, ALLOCATABLE :: JSR(:)
 C>        BUFR message.
```

Comment:
Could you please change the commented reference to the subroutine name from `rewnbf` to `rewnbf()`?  That way, doxygen will generate an automatic link to the documentation for that subroutine from this module.

Reply:
done

Reply:
Thanks Jack!